test/controllers/admin_controller_test.rb

```diff
@@ -282,47 +283,47 @@ def teardown; end
 
     # Test id param
     post :edit_user, params: { id: 'test', email: 'bad format', jurisdiction: new_jur.id,
-                               role_title: 'analyst', is_api_enabled: false, is_locked: false }, as: :json
+                               role_title: 'analyst', is_api_enabled: false, is_locked: false, notes: '' }, as: :json
     assert_response :bad_request
 
     # Test email param
     post :edit_user, params: { id: 17, email: 'bad format', jurisdiction: new_jur.id,
-                               role_title: 'analyst', is_api_enabled: false, is_locked: false }, as: :json
+                               role_title: 'analyst', is_api_enabled: false, is_locked: false, notes: '' }, as: :json
     assert_response :bad_request
 
     post :edit_user, params: { id: 17, jurisdiction: new_jur.id, role_title: 'analyst', is_api_enabled: false,
-                               is_locked: false }, as: :json
+                               is_locked: false, notes: '' }, as: :json
     assert_response :bad_request
 
     # Test bad jurisdiction param
     post :edit_user, params: { id: 17, email: 'test@testing.com', jurisdiction: '',
-                               role_title: 'analyst', is_api_enabled: false, is_locked: false }, as: :json
+                               role_title: 'analyst', is_api_enabled: false, is_locked: false, notes: '' }, as: :json
     assert_response :bad_request
 
     # Test invalid jurisdiction param (out of scope jurisdiction)
     post :edit_user, params: { id: 17, email: 'test@testing.com', jurisdiction: Jurisdiction.find_by(path: 'USA, State 2').id,
-                               role_title: 'analyst', is_api_enabled: false, is_locked: false }, as: :json
+                               role_title: 'analyst', is_api_enabled: false, is_locked: false, notes: '' }, as: :json
     assert_response :bad_request
 
     # Test role param
     post :edit_user, params: { id: 17, email: 'test@testing.com', jurisdiction: new_jur.id, role_title: 'test',
-                               is_api_enabled: false, is_locked: false }, as: :json
+                               is_api_enabled: false, is_locked: false, notes: '' }, as: :json
```

Comment:
can we add a test for the `notes` param

Reply:
Fixed in https://github.com/SaraAlert/SaraAlert/pull/805/commits/9e3e31175674140efbeb877b6bbbf711cb09768f